keras_cv/losses/simclr_loss.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow import keras
+
+
+class SimCLRLoss(tf.keras.losses.Loss):
+    """Implements SimCLR Cosine Similarity loss.
+
+    SimCLR loss is used for contrastive self-supervised learning.
+
+    Args:
+        temperature: a float value between 0 and 1, used as a scaling factor for cosine similarity.
+
+    References:
+        - [SimCLR paper](https://arxiv.org/pdf/2002.05709)
+    """
+
+    def __init__(self, temperature, **kwargs):
+        super().__init__(**kwargs)
+        self.temperature = temperature
+
+    def call(self, projections_1, projections_2):
```

Comment:
Need input / output description for `call`

Reply:
I've added some docs for this.